### DISCUSSION TITLE
Couldn't find package.json

### DISCUSSION THREAD

**Discussion by ghost** [warengonzaga/css-text-portrait-builder#24] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/73355252/142749905-1e461d60-e875-42cb-be28-e6f7f25b3bcb.png)

**Comment by warengonzaga** [comment, marked as answer]
You did not follow the correct [build steps](https://github.com/WarenGonzaga/css-text-portrait-builder#-building-steps). You need to make sure you meet the following [requirements](https://github.com/WarenGonzaga/css-text-portrait-builder#-requirements) before doing that.

Also, make sure that your current working directory is inside the builder directory. In your case, you are in the wrong directory. That's why you are seeing that error because there is no package.json file in your current working directory.

You can follow this video tutorial in tagalog: https://github.com/WarenGonzaga/css-text-portrait-builder/discussions/20 😎
